ci/docker/os-image-stemcell-builder-jammy/Dockerfile

```diff
@@ -56,6 +56,15 @@ RUN groupadd -o -g ${GROUP_ID} ubuntu                  \
   && useradd -u ${USER_ID} -g ${GROUP_ID} -m ubuntu \
   && echo 'ubuntu ALL=NOPASSWD:ALL' >> /etc/sudoers
 
+# install syft for sbom generation
+ADD syft.checksum /tmp/syft.checksum
+ADD https://raw.githubusercontent.com/anchore/syft/5a865d0d904b90252c02e0cceeac61c3d882837d/install.sh /tmp/install.sh
+RUN \
+    cd tmp \
+    sha256sum -c --status syft.checksum && \
+    sh install.sh -b /usr/local/bin && \
+    rm install.sh syft.checksum
```

Comment:
I would prefer a simple one-liner like: https://github.com/cloudfoundry/bosh-linux-stemcell-builder/blob/4481987e71b816ae0bf7f20119adc3aa7f610cf4/ci/docker/os-image-stemcell-builder-jammy/Dockerfile#L113-L116 over downloading and executing an external install script.